en/docs/reference/icp/observability-setup.md

```
@@ -355,23 +308,17 @@ The shipped `scripts/scripts.lua` provides these Lua functions used by the filte
 After the BI runtime has been running for a minute or two:
 
 ```bash
-curl -ku admin:<password> https://localhost:9200/_cat/indices?v
+curl http://localhost:9200/_cat/indices/ballerina-*?v
 ```
 
 You should see:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)